README.rst

```diff
@@ -68,6 +70,71 @@ Requirements
 If using postgres as database driver, ensure the `postgres contrib`_ package is
 installed on server side.
 
+Recomended usage
+================
+
+Given target minion_id is 'srv'.
+
+/srv/salt/top.sls::
+
+    base:
+        'srv'
+            - saemref
+            - saemref.supervisor
+
+
+/srv/pillar/top.sls::
+
+    base:
+        'srv':
+            - saemref
+
+
+/srv/pillar/saemref.sls::
+
+    saemref:
+        lookup:
+            instance:
+                base_url: http://saemref.example.com
+                # generated with: pwgen -sy 32
+                sessions_secret: "iajI*=^m;oqNc7:%ta'A6r3BDLUjB^,y"
+                authtk_session_secret: ")E6"LT,XMUd."Ekg+(<2m%RVqLxdV[a4"
+                authtk_persistent_secret: "Ay]%v4w>B+yP$|~BLTL^+785(6@=,Gk$"
```

Comment:
I'm not sure we should recommend non-alphanumeric characters. I had trouble with these in paste's INI parser recently. (/me anticipating migration)